src/tests/access/test_ownable_twostep.cairo

```diff
@@ -269,47 +279,49 @@ fn test_renounceOwnership_from_zero_address() {
 #[should_panic(expected: ('Caller is not the owner',))]
 fn test_renounceOwnership_from_nonowner() {
     let mut state = setup();
-    testing::set_caller_address(OTHER());
+    start_cheat_caller_address(test_address(), OTHER());
     state.renounceOwnership();
 }
 
 #[test]
 fn test_full_two_step_transfer() {
     let mut state = setup();
-    testing::set_caller_address(OWNER());
+    let mut spy = spy_events();
+    start_cheat_caller_address(test_address(), OWNER());
     state.transfer_ownership(OTHER());
 
-    assert_event_ownership_transfer_started(OWNER(), OTHER());
+    spy.assert_event_ownership_transfer_started(test_address(), OWNER(), OTHER());
     assert_eq!(state.owner(), OWNER());
     assert_eq!(state.pending_owner(), OTHER());
 
-    testing::set_caller_address(OTHER());
+    start_cheat_caller_address(test_address(), OTHER());
     state.accept_ownership();
 
-    assert_only_event_ownership_transferred(ZERO(), OWNER(), OTHER());
+    spy.assert_only_event_ownership_transferred(test_address(), OWNER(), OTHER());
     assert_eq!(state.owner(), OTHER());
     assert!(state.pending_owner().is_zero());
 }
 
 #[test]
 fn test_pending_accept_after_owner_renounce() {
     let mut state = setup();
-    testing::set_caller_address(OWNER());
+    let mut spy = spy_events();
+    start_cheat_caller_address(test_address(), OWNER());
```

Comment:
Very small detail, but since we are calling test_address quite a few time in this test, is probably good to move it into a contract_address local variable to avoid instantiating the ContractAddress each time.

Reply:
Introduced a local variable where `test_address()` is used more than once